lambdas/db-migration/src/migrations/20201014105058_create_granules_table.ts

```diff
@@ -6,6 +6,14 @@ export const up = async (knex: Knex): Promise<void> =>
       .bigIncrements('cumulusId')
       .comment('Internal Cumulus ID for a granule')
       .primary();
+    table
+      .text('granuleId')
+      .comment('Granule ID')
+      .notNullable();
+    table
+      .enum('status', ['running', 'completed', 'failed'])
+      .comment('Ingest status of the granule')
+      .notNullable();
     table
       .integer('collectionCumulusId')
       .references('cumulusId')
```

Comment:
I know this isn't really a part of your PR but I wanted to get some clarification. Is this setting `cumulusId` as a foreign key?

Reply:
Also, where would this `collectionCumulusId` value come from if `Collection` is a name and version?

Reply:
collectionCumlusId is a foriegn key referring to the cumulusId on the Collections table - it's an (incrementing) ID that's generated on insertion and is seperate from the name/version identifier.

Reply:
> where would this collectionCumulusId value come from if Collection is a name and version?

That's where `getMessageCollectionCumulusId` comes in. It looks up the collection name/version from a workflow message in RDS, returning its `cumulusId`. Then, we use that `cumulusId` to set `collectionCumulusId` on granules, PDRs, and executions.

https://github.com/nasa/cumulus/pull/1987/files#diff-bc96e8e9a9c08e28b506ef0387539a0fc75bb657b28dd974c4419818a5e72607R85